jina/orchestrate/flow/base.py

```diff
@@ -327,6 +327,30 @@ def __init__(
 
         """Create a Flow. Flow is how Jina streamlines and scales Executors.
 
+        EXAMPLE USAGE
+
+            Python API
+
+            .. code-block:: python
+
+                from jina import Flow
+
+                f = Flow().add(uses=MyExecutor)  # create Flow and add Executor
```

Comment:
```suggestion
                f = Flow().add(uses='jinahub+docker://SimpleIndexer')  # create Flow and add Executor
```
I think that all of the doc string example should be runable, the user should be able to copy paste it